src/Orneholm.CognitiveWorkbench.Web/Views/Vision/ComputerVision.cshtml

```diff
@@ -281,8 +287,60 @@
             </section>
         }
 
+        @if (Model.IsAnalyzed && Model.ComputerVisionAnalyzeResponse?.RecognizeTextOperationResult != null)
+        {
+            <section class="cwb-result-computervision-analyze">
+                <h2 class="mt-5">Computer Vision - Recognize Text</h2>
+
+                <div class="row">
+                    <div class="col-md-7">
+                        <div class="card mb-3 cwb-result-image">
+                            <img src="@Model.ComputerVisionAnalyzeResponse.ImageInfo.Url" class="card-img-top" alt="@Model.ComputerVisionAnalyzeResponse.ImageInfo.Description" />
+                            <div class="cwb-image-boxes">
+                                <div class="cwb-image-boxes-ocr-lines">
```

Comment:
Change to `cwb-image-boxes-recognizetext-lines`.
It collides with the classnames of the ocr now.

Reply:
Changing it in an incoming update.